### DISCUSSION TITLE
Want site similar to minimal mistakes site

### DISCUSSION THREAD

**Discussion by techtanic** [mmistakes/minimal-mistakes#2881] [Q&A] [answered]
I want something similar the this page https://mmistakes.github.io/minimal-mistakes/
i tried it but it wont work
https://techtanic.github.io/

**Comment by iBug** [comment, marked as answer]
Can you at least show what you have tried? It's hard to answer without any context.

**Reply by techtanic** [reply]
Sorry to say, I realized that using HTML is better than Jekyll. 😞

**Reply by iBug** [reply]
Then there's not much we can help you with. The whole website, including its [HTML templates](https://github.com/mmistakes/minimal-mistakes/tree/master/_layouts) and [CSS](https://github.com/mmistakes/minimal-mistakes/tree/master/_sass), is open-source under the MIT license. With a certain level of knowledge with these front-end technologies, you will be able to reconstruct everything onto your website.
